src/main/java/com/google/sps/util/Validator.java

```diff
@@ -7,9 +7,7 @@ public class Validator {
 
     /**
      * @param String username from post request
-     * @return  boolean, true if String is an acceptable username
-     *          containing: letters, digits, and no symbols
-     * may be updated later
+     * @return  
```

Comment:
not sure but did you not intend to delete the comments?

Reply:
I did because the description was no longer accurate.